apiservers/engine/backends/network.go

```diff
@@ -114,7 +114,7 @@ func (n *Network) CreateNetwork(name, driver string, ipam apinet.IPAM, options m
 		IPAM:      pools,
 	}
 
-	created, err := PortLayerClient().Scopes.Create(scopes.NewCreateParams().WithConfig(cfg))
+	created, err := PortLayerClient().Scopes.CreateScope(scopes.NewCreateScopeParams().WithConfig(cfg))
```

Comment:
can you raise an issue against go-swagger for that - and a comment in the code as to why the name, referencing that issue.